auth/src/main/res/values/colors.xml

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="authui_colorPrimary">#3E50B4</color>
-    <color name="authui_colorPrimaryDark">#2F3E9E</color>
-    <color name="authui_colorAccent">#3E50B4</color>
-    <color name="authui_inputTextColor">#FFFFFF</color>
-    <color name="authui_inputTextColorDark">#000000</color>
+    <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorAccent">#FF4081</color>
```

Comment:
This is where all the magic happens: the app's theme overrides this so we get theme customization for free. However, we need to keep these values to be able to build the `auth` module.